docs/source/_static/versions.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "name": "latest (main)",
```

Comment:
You should not only include main branch, all versions associated with tags in this repo should be added to this json file.